src/ticketmap/index.tsx

```diff
@@ -251,15 +259,15 @@ export default class TicketMap extends Component<Props, State> {
     return this.fillSection(sectionId, shouldHighlight)
   }
 
-  selectSection(section: string) {
+  selectSection = (section: string) => {
```

Comment:
What is this change for?

Reply:
I was getting errors that functions weren't bound... Some of these functions were called outside the context of the component (via the public API), and I remember you telling me that fat arrow functions don't require explicit binding to the component. I went through and did this for a bunch of functions related to the API just to be safe.

Reply:
Ah, I see!